src/aiodynamo/credentials.py

```diff
@@ -198,7 +203,9 @@ async def get_key(self, http: HTTP) -> Optional[Key]:
                 self._refresher = asyncio.create_task(self._refresh(http))
             else:
                 logger.debug("already refreshing")
-        return self._metadata and self._metadata.key
+        if self._metadata:
+            return self._metadata.key
+        return None
```

Comment:
Does `mypy` force you to do this? 😱

Reply:
it didn't understand the `and` shortcut 😭